examples/lightrag_api_oracle_demo..py

```diff
@@ -81,7 +81,7 @@ async def get_embedding_dim():
 
 async def init():
     # Detect embedding dimension
-    embedding_dimension = await get_embedding_dim()
+    embedding_dimension = 1024  # await get_embedding_dim()
```

Comment:
Should we keep `await get_embedding_dim()` here?

Reply:
It's for debug, should be removed. I have fixed follow your advice.